pallets/asset-manager/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-asset-manager"
+authors = [ "Parallel Team" ]
+edition = "2021"
+version = "0.1.0"
+
+[dependencies]
+log = { version = "0.4", default-features = false }
+serde = { version = "1.0.124", optional = true }
+
+# Primitives
+parallel-primitives = { path = "../../primitives", default-features = false }
+
+# Substrate
+frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
```

Comment:
Please fix formatting and use single quote to make it consistent like other pallets. Please refer this https://github.com/parallel-finance/parallel/blob/master/pallets/amm/Cargo.toml for example